scripts/tasks/build-codepen-examples.js

```diff
@@ -0,0 +1,52 @@
+module.exports = function(options) {
+  const path = require('path');
+  const transformer = require('./codepen-examples-transform');
+  const glob = require('glob');
+  const files = glob.sync(path.resolve(process.cwd(), 'src/components/**/examples/*Example*.tsx'));
+  const jscodeshift = require('jscodeshift');
+  const fs = require('fs');
+  const async = require('async');
+
+  return processFiles();
+
+  function processFiles() {
+    const promises = [];
+    if (files.length) {
+      promises.push(
```

Comment:
I think you don't need this promises variable anymore - you should just return that `new Promise` below